finesse/hardware/plugins/spectrometer/opus_interface_base.py

```diff
@@ -17,13 +15,24 @@ def from_response(cls, errcode: int, errtext: str) -> OPUSError:
         return cls(f"Error {errcode}: {errtext}")
 
 
-class OPUSInterfaceBase(Device, name=SPECTROMETER_TOPIC, description="OPUS device"):
+class OPUSInterfaceBase(SpectrometerBase):
```

Comment:
I had not relealised of this, but I have two questions:
1. Is it possible (i.e. does it work as intended) having an `abstractmethod` (`request_command`) in a non-abstract class? 
2. What is the point of this intermediate class now that we have the `SpectrometerBase`?  All of these methods could go in the `OpusInterface`, deriving directly from `SpectrometerBase`, and then have the `DummyOPUSInterface` inheriting from `OpusInterface`. To be honest, that is something we could have done before, but I just thought about it now.

Reply:
1. Yep! It's still an abstract class (i.e. you can't create an instance of it).
2. This class doesn't do *much*, just sends the appropriate command to the OPUS device. The `DummyOPUSInterface` is pretty similar to `OPUSInterface` in that it accepts the same commands, has the same states and throws the same errors. We could drop the `OPUSInterfaceBase` class, but then we'd basically just end up moving the boilerplate from it to `OPUSInterface` and `DummyOPUSInterface` instead. Not sure.

Reply:
I'm going to merge this for now, if that's ok. V happy to revisit this discussion, but I'd like to make the release for FINESSE. We can always change this later.

Reply:
Not a problem!